tastypie_swagger/urls.py

```diff
@@ -1,13 +1,29 @@
+from django import __version__ as django_version
 try:
 	from django.conf.urls import patterns, include, url
 except ImportError:
 	from django.conf.urls.defaults import patterns, include, url
 
 from .views import SwaggerView, ResourcesView, SchemaView
 
-urlpatterns = patterns('',
-    url(r'^$', SwaggerView.as_view(), name='index'),
-    url(r'^resources/$', ResourcesView.as_view(), name='resources'),
-    url(r'^schema/(?P<resource>\S+)$', SchemaView.as_view()),
-    url(r'^schema/$', SchemaView.as_view(), name='schema'),
-)
+django_version = django_version.split('.')
+main_ver = int(django_version)[0]
+secondary_ver = int(django_version)[1]
+
+if main_ver == 1 and secondary_ver < 9:
+	urlpatterns = patterns('',
```

Comment:
Something something DRY. Declare `urlpatterns` as a list and then if the version check is earlier than 1.8 (which is when it was deprecated) wrap it in `patterns('', *urlpatterns)`